### PR TITLE
Stop using “brackets” to mean “braces”

### PR DESCRIPTION
**Pull request by adiabatic**
I’ve been tripped up repeatedly by how the vocabulary gives me `{}` when I say “brackets”. In my part of the world, at least, “brackets” are `[]` and “braces” are `{}`. “Square brackets” is a belt-and-suspenders redundancy, like “ATM machine” or “PIN number”.

For what it's worth, aegis [agreed with me in Slack](https://talonvoice.slack.com/archives/C7ERD5Q5T/p1703888321381209) that there ought to be only one way to refer to these sorts of things.

I looked through [old Issues](https://github.com/talonhub/community/issues?q=is%3Aissue+bracket) and [old PRs](https://github.com/talonhub/community/pulls?q=is%3Apr+bracket) and nothing explicitly said “we choose to mean that ‘brackets’ are either curly or square depending on whether you say ‘square’ first.”

### CODE REVIEW

**Comment by auscompgeek** [comment]
FWIW, in my part of the world, brackets usually refer to parentheses (or round brackets), curly brackets refer to braces, and square brackets are, well, square.

**Review by knausj85 — APPROVED** [review]
I'm personally fine with this change, but I'm going to leave it open for others to poke at.

File: plugin/symbols/symbols.talon [hunk 1]
```diff
@@ -12,19 +12,19 @@ empty escaped (dub string | dub quotes): user.insert_between('\\"', '\\"')
 empty string: user.insert_between("'", "'")
 empty escaped string: user.insert_between("\\'", "\\'")
 (inside parens | args): user.insert_between("(", ")")
-inside (squares | square brackets | list): user.insert_between("[", "]")
-inside (bracket | braces): user.insert_between("{", "}")
+inside (squares | brackets | list): user.insert_between("[", "]")
```
Comment (nriley) [inline plugin/symbols/symbols.talon:15]: I'd recommend keeping "square brackets" here.

File: plugin/symbols/symbols.talon [hunk 2]
```diff
@@ -12,19 +12,19 @@ empty escaped (dub string | dub quotes): user.insert_between('\\"', '\\"')
...
+inside braces: user.insert_between("{", "}")
 inside percent: user.insert_between("%", "%")
 inside (quotes | string): user.insert_between("'", "'")
 inside (double quotes | dub quotes): user.insert_between('"', '"')
 inside (graves | back ticks): user.insert_between("`", "`")
 angle that:
     text = edit.selected_text()
     user.paste("<{text}>")
-(square | square bracket) that:
+(square | bracket) that:
```
Comment (nriley) [inline plugin/symbols/symbols.talon:24]: Same — keep "square bracket" here.

**Review by nriley — COMMENTED** [review]
Thanks for doing this — I made many of these changes years ago in my own repo.

I think adding "curly bracket" everywhere we use "brace" would make sense too. That should cover all the common names!

File: plugin/symbols/symbols.talon (hunk 2 above)
Comment (adiabatic) [reply plugin/symbols/symbols.talon]: Done in https://github.com/talonhub/community/pull/1344/commits/f0dbeae39e4f65a7b00fb070efa896af4f805cc0.

File: plugin/symbols/symbols.talon (hunk 1 above)
Comment (adiabatic) [reply plugin/symbols/symbols.talon]: Done in https://github.com/talonhub/community/pull/1344/commits/f0dbeae39e4f65a7b00fb070efa896af4f805cc0.

**Comment by adiabatic** [comment]
Updated with @nriley's suggestions.

There are "squares" and "braces" in [text_navigation.py, in `navigation_target_names`](https://github.com/talonhub/community/blob/c6e1969ab516d509f51b9c8acd01bd35d2262d28/plugin/text_navigation/text_navigation.py#L43). I'm not sure what, if anything, to do with those.

**Comment by nriley** [comment]
I think it's OK to leave text_navigation alone as at least it doesn't misuse "brackets" — I don't think it gets a lot of use (personally I keep forgetting it exists / to use it!). In an ideal world we could harmonize everything with Cursorless…

**Pull request merged**